Add synchronous replication docs (#4207)
pingcap/docs: pd-configuration-file.md

+ Determines whether to enable the telemetry collection feature in TiDB Dashboard.
+ Default value: `true`
+ See [Telemetry](/telemetry.md) for details.

## `replication-mode`

> Configuration items related to the replication mode of all Regions. See [Enable synchronous replication in PD configuration file](/synchronous-replication.md#enable-synchronous-replication-in-pd-configuration-file) for details.

Author: ```suggestion
Configuration items related to the replication mode of all Regions. See [Enable synchronous replication in PD configuration file](/synchronous-replication.md#enable-synchronous-replication-in-the-pd-configuration-file) for details.
```

> Configuration items related to the replication mode of a single Region. See [Enable synchronous replication in PD configuration file](/synchronous-replication.md#enable-synchronous-replication-in-pd-configuration-file) for details.

Reviewer: It works for all regions instead of single one.

> Configuration items related to the replication mode of a single Region. See [Enable synchronous replication in PD configuration file](/synchronous-replication.md#enable-synchronous-replication-in-pd-configuration-file) for details.

Author: ```suggestion
Configuration items related to the replication mode of all Regions. See [Enable synchronous replication in PD configuration file](/synchronous-replication.md#enable-synchronous-replication-in-pd-configuration-file) for details.
```
